collector/src/main/java/com/bitmovin/analytics/adapters/PlayerAdapter.java

```diff
@@ -3,13 +3,13 @@
 import com.bitmovin.analytics.data.DRMInformation;
 import com.bitmovin.analytics.data.EventData;
 
-import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 public interface PlayerAdapter {
     EventData createEventData();
 
     void release();
     long getPosition();
-    @NotNull
+    @Nullable
```

Comment:
`com/bitmovin/analytics/exoplayer/ExoPlayerAdapter.java` still you wrong `@NotNull` annotation